strapi/strapi
# chore(database): use naming functions for identifiers in metadata

### What does it do?

Refactors database/metadata to use centralized naming functions

### Why is it needed?

To ensure all metadata is coming from a centralized place so that we can enable shortening on identifier tokens and support names of arbitrary length in any database

### How to test it?

Absolutely nothing should logically change; all database identifiers should still be the same as before.

- a strapi project full of data without this PR should be able to switch to this PR and not lose any data or notice any difference
- once #19560 is merged into v5/main and then into here, those tests should pass unchanged (that is, proving the above, that nothing has changed in the db identifiers)
  
### Related issue(s)/PR(s)

Blocked by https://github.com/strapi/strapi/pull/19560 to be merged into `v5/main` as validation that the tests pass there, and then also pass here without any change -- demonstrating that nothing has actually changed in terms of the logic and output.

Blocked by https://github.com/strapi/strapi/pull/19460 which initializes the naming functions
